pkg/components/ebpf/common/kafkaparser/common.go

```diff
@@ -0,0 +1,307 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package kafkaparser
+
+import (
+	"encoding/binary"
+	"errors"
+)
+
+const (
+	Int8Len            = 1
+	Int16Len           = 2
+	Int32Len           = 4
+	Int64Len           = 8
```

Comment:
question: should we explicitly have: MessageSizeLen, APIKeyLen, etc. so we can avoid having the base type lengths and comments?

Reply:
i did it intentionally actually to kind of describe that the MinKafkaRequestLen is comprised of the length of combined fields, thought it might be more descriptive
but if you think its confusing i can remove this

Reply:
no, it's not confusing at all for me 👍🏻